src/core/server/saved_objects/saved_objects_service.test.ts

```diff
@@ -142,6 +143,7 @@ describe('SavedObjectsService', () => {
 
     describe('#registerType', () => {
       it('registers the type to the internal typeRegistry', async () => {
+        // we mocked registerCoreObjectTypes above, so this test case only reflects direct calls to the registerType method
```

Comment:
Can you also add a test to assert `registerCoreObjectTypes(this.typeRegistry);` is called during `setup`?

Reply:
Done in 42cc694.